<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TokenDialog</class>
 <widget class="QDialog" name="TokenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add or Modify Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 120, 120);
color:white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Reference Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameBox">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(50,50,50);
color:white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Token</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tokenBox">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(50,50,50);
color:white;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Bot?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>TokenDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>TokenDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
